pkg/cache/cache.go

```diff
@@ -39,3 +41,43 @@ func RemoteCacheEnabled() (bool, error) {
 	}
 	return false, nil
 }
+
+func AddCache(bucketName string, region string) error {
```

Comment:
Can we rename this as technically this is only for adding a remote S3 cache?

Maybe `AddRemoteS3Cache` or `AddRemoteCache` ?